### ISSUE TITLE
Help nominate network members for recognition

### ISSUE THREAD

**Issue by OpenMatt** [MozillaFoundation/Story-Engine#16]
- pursuant to conversation with Rizwan. They're going to identify the  top-performing members of the network and send them a recognition letter
- we should make sure we log and selectively interview many of them

**Comment by prefontaine** [comment]
@OpenMatt assuming that this will move forward once we touch base with Rizwan.

**Comment by OpenMatt** [comment]
yep -- we're blocked on Rizwan for this. Nothing to do yet.

**Comment by OpenMatt** [comment]
Matt will nudge Rizwan on this one

**Comment by OpenMatt** [comment]
I asked Rizwan for an update. Let's assume this won't happen this month and then re-sched as we know more.

**Comment by OpenMatt** [comment]
Update from Rizwan:

We are pushing the teams hard to get their nominations in ASAP so that we can then triage and begin to finalize the shortlists. The pace of the nomination worries me but we should hopefully have all nominations in latest by the final week of Sept. We can think of sharing names by mid-Oct or so.

**Comment by OpenMatt** [comment]
suggestion from @chrislarry33 : take some of the stand-out interviews we've done and nominate those people into Rizwan's recognition queue. 

(e.g., instead of us waiting for him, we can also help feed into his work and process there.)

**Comment by OpenMatt** [comment]
@prefontaine do you have any early "top 3" or "top 5" people you've interviewed so far?

**Comment by OpenMatt** [comment]
cc @rtufail

**Comment by prefontaine** [comment]
@OpenMatt @rtufail @chrislarry33 - I'd say Mmaki Jantjies for sure. Let me think on others.

**Comment by OpenMatt** [comment]
@rtufail has a Google Form set up for this -- maybe he can drop it here and @prefontaine can fill it in.

**Comment by OpenMatt** [comment]
Key learning to file here for our next Heartbeat meeting: merging these two pieces of work is a priority going forward.

**Comment by rtufail** [comment]
Thanks for your help on this. The form is at: https://docs.google.com/a/mozillafoundation.org/forms/d/1xo0CHYiox1vygzs7M_WOI3-PdZXVOMw6uQIhiM327kE/

**Comment by OpenMatt** [comment]
Christine, do you think you could make 3 - 5 nominations using that form based on the interviews you’ve already done?

**Comment by prefontaine** [comment]
Yes. Will do!

**Comment by rtufail** [comment]
Hi Christine, would you be able to nominate a few today, and possibly a timeline for the others? We did communicate an original deadline of today (9/9) to staff but will be extending. Thanks for your help on this - greatly appreciated.

**Comment by prefontaine** [comment]
I am doing that now. But a lot of the questions I cannot answer. Either
because I don't know interviewees interactions with Mozilla staff or
because the only input I have is one interview. I will just submit what I
know and send you the transcripts once I receive them. For the scales at
the end I'm totally not the person to fill those out. I'm just putting 10
for now and you can follow up with the appropriate staff member.

**Comment by rtufail** [comment]
Great, thanks

**Comment by OpenMatt** [comment]
That all makes perfect sense, Christine -- thanks for diving in with
limited information. Helpful!

**Comment by prefontaine** [comment]
@rtufail I've submitted three and just need a bit of time to think on the rest. May be a few more.

**Comment by chrislarry33** [comment]
huge thanks @prefontaine and @OpenMatt for adding! We can make sure any gaps in application are filled in

**Comment by rtufail** [comment]
Excellent - thanks

**Comment by rtufail** [comment]
@prefontaine and @OpenMatt, I know we had spoken about how the nomination process can also help accelerate your work. Feel free to take a look at this link (https://docs.google.com/spreadsheets/d/1wOW6gzIXVZBHfPsqRGHLwKqMVZXGpKJVQeX5j53_8wM/edit#gid=0), which has the current list of all people who have been nominated so far. Might spark ideas about some names. I can also share the nomination info details, if that would be useful. Thanks

**Comment by OpenMatt** [comment]
@rtufail one question we have about this is: what's the criteria in terms of their direct affiliation with / belonging to the Mozilla universe? 
- e.g., some people are clearly aligned / all-in on Mozilla (e.g., they're Club Captains, have been volunteers for years, etc.) 
- others are clearly aligned with our _issues_, but less aligned with Mozilla. 

(Christine and I were talking about this breaks down into a 4-quadrant axis -- 
alignment w. Mozilla on x axis, alignment on issues y)

**Comment by rtufail** [comment]
Great question, @OpenMatt 

We are open to both kind of nominations. One of the objectives of the R/R strategy is to intentionally develop that sense of membership, and to draw people - who might already be aligned with the issues - more formally into the Mozilla 'universe'.

I think mapping people on the matrix is great. If we imagine alignment to Mozilla on the horizontal and alignment to issues on the vertical, then ideal scenario is to move many people from the top left to the top right. These people already care about the issues, and are doing something about them. Helping them see how Mozilla is a great 'platform' to amplify their impact would/could be a huge win.

**Comment by prefontaine** [comment]
Thanks for the clarification. Will post more today and explain where I
think the person falls in the matrix :)

**Comment by OpenMatt** [comment]
I spoke to @prefontaine -- she will review the list of existing nominees and see if she has more to add.

**Comment by OpenMatt** [comment]
I spoke to @prefontaine -- she will review the existing list and add others if she can.

**Comment by prefontaine** [comment]
@rtufail Matt sent me the names — but could you please give me access to the spreadsheet? I will also let you know when there are people that I "+1" — already see a few of those on there.

**Comment by rtufail** [comment]
@prefontaine - Sure. can you drop me an email at rizwan@mozillafoundation.org

**Comment by OpenMatt** [comment]
@rtufail where are we on this? Do you still need more nominations from @prefontaine -- or do you hav enough at this point?

**Comment by rtufail** [comment]
We have enough at this point. It would help if @prefontaine could provide additional feedback on a couple of the nominations, though. Have emailed here already ... will remind now

**Comment by prefontaine** [comment]
@rtufail I will provide that feedback -- apologies was unable to get to this while in India.

**Comment by rtufail** [comment]
@prefontaine a gentle nudge. Looking to close off all the nomination data soon so your feedback on the folks we discussed would be great

**Comment by prefontaine** [comment]
Thanks for the nudge @rtufail. I added on the 3 you mentioned. Please let me know if you need anything more.

**Comment by rtufail** [comment]
excellent .. thanks

**Comment by OpenMatt** [comment]
marking this done! great work Christine.
